tests/PHPStan/Rules/Classes/DuplicateDeclarationRuleTest.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Classes;
+
+use PHPStan\Rules\Rule;
+use PHPStan\Testing\RuleTestCase;
+
+class DuplicateDeclarationRuleTest extends RuleTestCase
+{
+
+	protected function getRule(): Rule
+	{
+		return new DuplicateDeclarationRule();
+	}
+
+	public function testDuplicateDeclarations(): void
+	{
+		$this->analyse(
+			[
+				__DIR__ . '/data/duplicate-declarations.php',
+			],
```

Comment:
the indentation looks wrong

Reply:
are you sure? it's the same as this one locally: https://github.com/phpstan/phpstan-src/blob/master/tests/PHPStan/Rules/Classes/ClassConstantRuleTest.php

Reply:
Oh sorry, you're right. Anyway, look at the failing build. The files can be excluded from lint like this https://github.com/phpstan/phpstan-src/blob/6425cdfc457611b9a5a526ecd700e690b33108f6/build.xml#L118-L150

Reply:
Thanks. I noticed it had failed but I wasn't sure what to do about it.